common/app/common/JsMinifier.scala

```diff
@@ -19,11 +19,10 @@ object JsMinifier {
     /* Checks */
     options.setCheckTypes(true)
     options.setCheckSuspiciousCode(true)
-    options.setReportMissingOverride(CheckLevel.WARNING)
-    options.setCheckProvides(CheckLevel.WARNING)
-    options.setCheckGlobalNamesLevel(CheckLevel.WARNING)
-    options.setBrokenClosureRequiresLevel(CheckLevel.WARNING)
```

Comment:
Deleted in https://github.com/google/closure-compiler/commit/f8b28e766b9ccb3600ca07b8319ed04a0c7404b0